_includes/main_scripts.html

```diff
@@ -69,3 +69,7 @@
 <script src="{{- site.baseurl -}}/assets/js/google_analytics.js"></script>
 <script async src="https://www.google-analytics.com/analytics.js"></script>
 <script async src="{{- site.baseurl -}}/assets/js/autotrack.js"></script>
+
+{%- comment -%} Isomer's Google Tag (used for Google Analytics){%- endcomment -%}
```

Comment:
Nit: lets rename this comment to `Isomer's Google Tag GA4` since we have a comment above that states `Google Analytics`. 
I am not sure sure if this messes up anything (can resolve if this does) could we rename the previous `{%- comment -%} Google Analytics {%- endcomment -%}` to `{%- comment -%} Google Analytics UA{%- endcomment -%}` for clarity?

Reply:
Yeah that makes sense, https://github.com/isomerpages/isomerpages-template/pull/294/commits/a799117de118d6c4978d77b76bfaa61cb040852f